--- a/Makefile
+++ b/Makefile
@@ -97,7 +97,7 @@ LIBDIRS	:= $(CTRULIB)
 #---------------------------------------------------------------------------------
 export INCLUDE	:=	$(foreach dir,$(INCLUDES),-I$(CURDIR)/$(dir)) \
 			$(foreach dir,$(LIBDIRS),-I$(dir)/include) \
-			-I$(CURDIR)/$(BUILD)
+			-I$(CURDIR)/$(BUILD) \
+			-I/opt/devkitpro/portlibs/3ds/include
 
 export LIBPATHS	:=	$(foreach dir,$(LIBDIRS),-L$(dir)/lib)
+export LIBPATHS	+=	-L/opt/devkitpro/portlibs/3ds/lib
